Observatory-Control-System/Non-standard-Operations/Computer-Room-Shutdown/index.rst

```diff
@@ -48,10 +48,14 @@ The following is the list of safe to power-off servers. The computers can be pow
 
 **IT**
 
-- vsphere[2-3].cp.lsst.org - Previous movement of VMs to vsphere01.cp.lsst.org
+*snapshot and shutdown VMs first*
+- vsphere[2-3].cp.lsst.org - *hvaccp* and *dccp1* VMs migrate to vsphere01.cp.lsst.org  . Maintenance mode first then shut down from  vcenter.cp.lsst.org/ui
```

Comment:
In this section, we first need to shut down the VMs running on the specific vsphere node. This should speed up the process when we enter the vpsphere host into maintenance mode prior to shutting it down.